openshift_scalability/README.md

```diff
@@ -104,6 +104,7 @@ tuningsets:
 > * The ***Tuning parameters*** have following function:
 >  * ***stepping*** : This feature makes sure that after each "stepsize" pod/template requests are submitted, they enter the "Running" state. After all the pods/templates in the given step are Running, then there is a delay = "pause" , before the next step.
 >  * ***rate_limit*** : This makes sure that there is a delay of "rate_limit.delay" between each pod/template request submission.
+>  * ***--tolerate-bad-pods*** : This option ensures that cluster-loader utility won't wait till all the pods would come to running state. This is useful when we intentionally give bad pods which are bound to fail. By default the utility waits till all the pods come to running state.
```

Comment:
This is not a tuning parameter, please put it in the correct section of the README.